src/openforms/registrations/contrib/stuf_zds/tests/test_backend.py

```diff
@@ -695,3 +694,15 @@ def test_plugin(self, m, mock_task):
                 "//zkn:object/zkn:formaat": "application/msword",
             },
         )
+
+    def test_no_reference_can_be_extracted(self, m):
```

Comment:
Name of the test is wrong, as a reference *can* be extracted here.

Reply:
nice catch, fixed!